lib/cookie.js

```diff
@@ -885,7 +899,7 @@ CookieJar.prototype.setCookie = function(cookie, url, options, cb) {
 
   // S5.3 step 1
   if (!(cookie instanceof Cookie)) {
-    cookie = Cookie.parse(cookie);
+    cookie = Cookie.parse(cookie, !options || !options.loose);
```

Comment:
Should this pass `{loose: options.loose}` or maybe just `options` entirely? On a hunch this might require code coverage -- try adding `jar.setCookieSync('=a', url)` and `jar.setCookieSync('=a',url,{loose:true})` and make sure those behave as expected?